src/metabase/models/revision.clj

```diff
@@ -45,15 +45,17 @@
       {:before before
        :after  after})))
 
-(defmulti diff-str
-  "Return a string describing the difference between `object-1` and `object-2`."
+(defmulti diff-strs
```

Comment:
we want this to be a seq of strings now so we can infer `has-multiple-changes` in https://github.com/metabase/metabase/blob/4f15c36457384187ae7fc8271be6fa99e50956ba/src/metabase/models/revision.clj#L111